app/views/foods/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="form_container">
+<h1>Add Food</h1>
+<%= form_with model: @food, class: "form" do |form| %>
+  <div class="form-row">
+  <h3>Name:</h3>
+  <%= form.text_field :name, placeholder: "Add Name", required: true, class: "text_field" %>
+  </div>
+  <div class="form-row">
+  <h3 class="select_box">Measurement unit:</h3>
+  <%= form.select :measurement_unit, ['kg', 'lb', 'oz', 'l', 'g', 'pz'], required: true, class: "select_box" %>
```

Comment:
### OPTIONAL

- [ ] It will be good if you update the style of the unit field. They do not seem.

![image](https://user-images.githubusercontent.com/38635158/181622280-c4e7924b-700e-4e3a-b6b5-ad9cafce00c3.png)